docker/docker-compose.development.yml

```diff
@@ -1,21 +1,21 @@
 version: '3'
 services:
-   ganache:
-     restart: always
-     build:
-       context: ./development
-       dockerfile: ganache.dockerfile
-     env_file: ./development/.env
-     ports:
-       - 8545:8545
-   jaeger:
-      restart: always
-      image: jaegertracing/all-in-one:1.13
-      ports:
-        - '5775:5775/udp'
-        - '6831:6831/udp'
-        - '6832:6832/udp'
-        - '5778:5778'
-        - '16686:16686'
-        - '14268:14268'
-        - '9411:9411'
\ No newline at end of file
+  ganache:
+    restart: always
+    build:
+      context: ./development
+      dockerfile: ganache.dockerfile
+    env_file: ../.env.dev.local
+    ports:
+      - 8545:8545
+  jaeger:
+    restart: always
+    image: jaegertracing/all-in-one:1.12
```

Comment:
Was this intended to be a change in image version here?

Reply:
Oups!